podman/api_connection.py

```diff
@@ -0,0 +1,108 @@
+""" Provides a Connection to a Podman service. """
+import socket
+import urllib.parse
+from contextlib import AbstractContextManager
+from http import HTTPStatus
+from http.client import HTTPConnection
+
+import podman.errors as errors
+import podman.images as images
+import podman.system as system
+
+
+class ApiConnection(HTTPConnection, AbstractContextManager):
+    """ ApiConnection provides a specialized HTTPConnection
+        to a Podman service."""
```

Comment:
Howcome the extra whitespace on the left-hand side?  (new to me)

Reply:
Making the docstring left aligned? Easier to read.